src/main/scala/com/tapad/docker/DockerComposePlugin.scala

```diff
@@ -107,6 +109,26 @@ class DockerComposePluginLocal extends AutoPlugin with ComposeFile with DockerCo
         stopRunningInstances(state, args)
     }
 
+  lazy val dockerComposeRestartCommand = Command.args("dockerComposeRestart", ("dockerComposeRestart", "Restarts a local Docker Compose instance."),
```

Comment:
You should also be able to add some unit test cases for some of this new functionality. Check out the `InstanceStoppingSpec`